pkgs/by-name/dh/dhcpig/package.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dhcpig";
+  version = "1.6";
+
+  src = fetchFromGitHub {
+    owner = "kamorin";
+    repo = "DHCPig";
+    rev = version;
+    hash = "sha256-MquLChDuJe3DdkxxKV4W0o49IIt7Am+yuhdOqUqexS8=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    setuptools
+    scapy
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    scapy
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -R pig.py $out/bin/dhcpig
```

Nit: `-R` is not needed when copying a single file.

```suggestion
    cp pig.py $out/bin/dhcpig
```